server/src/main/java/org/apache/iotdb/db/consensus/statemachine/SchemaRegionStateMachine.java

```diff
@@ -47,7 +66,28 @@ public void stop() {}
   @Override
   protected TSStatus write(FragmentInstance fragmentInstance) {
     logger.info("Execute write plan in SchemaRegionStateMachine");
-    return new TSStatus(TSStatusCode.SUCCESS_STATUS.getStatusCode());
+
+    // Take create Timeseries as an example
+    PlanNode planNode = fragmentInstance.getFragment().getRoot();
+    // TODO transfer to physicalplan as a tentative plan
+    PhysicalPlan physicalPlan = planNode.transferToPhysicalPlan();
+    return write(physicalPlan);
+  }
+
+  protected TSStatus write(PhysicalPlan plan) {
+    boolean isSuccessful;
+    try {
+      isSuccessful = executor.processNonQuery(plan);
+    } catch (QueryProcessException e) {
+      logger.error("{}: query process Error: ", IoTDBConstant.GLOBAL_DB_NAME, e);
+      return RpcUtils.getStatus(e.getErrorCode(), e.getMessage());
```

Comment:
Should we invoke the interfaces provided by SchemaRegion directly?

Reply:
fix.